runtime/kusama/src/weights/pallet_elections_phragmen.rs

```diff
@@ -147,12 +147,10 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
 	// Storage: PhragmenElection Candidates (r:1 w:0)
 	// Storage: Balances Locks (r:250 w:250)
 	// Storage: System Account (r:250 w:250)
-	fn clean_defunct_voters(v: u32, d: u32, ) -> Weight {
+	fn clean_defunct_voters(v: u32, _d: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 42_000
-			.saturating_add((47_366_000 as Weight).saturating_mul(v as Weight))
 			// Standard Error: 40_000
-			.saturating_add((240_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add((48_208_000 as Weight).saturating_mul(v as Weight))
```

Comment:
Big jump up of weight value

Reply:
The git diff is a bit confusing here, but parameter `v` is nearly the same as before.  
Paramete `d` got kicked out. This is only a very small increase.